tests/src/Kernel/GraphQLTestBase.php

```diff
@@ -70,6 +70,7 @@ protected function setUp(): void {
     $this->installEntitySchema('graphql_server');
     $this->installEntitySchema('configurable_language');
     $this->installConfig(['language']);
+    $this->installEntitySchema('menu_link_content');
```

Comment:
On the query run there was an error logged. Something like "DB error: menu_link_content table does not exist". I also did not want to know why it happens 😅 So I just installed `menu_link_content` schema.

Reply:
Do we know in what test this was? This could indicate a hidden dependency that I don't think exists. So in theory this could even be flagging a breaking change.

(It's also not a module I'd expect should be added for all tests, so I'm not sure if `GraphQLTestBase` is the way to go).

Reply:
It was happening in the new test I created.